src/sensor/settings.py

```diff
@@ -326,3 +327,17 @@
     import raven
 
     RAVEN_CONFIG = {"dsn": SENTRY_DSN, "release": raven.fetch_git_sha(REPO_ROOT)}
+
+# OAuth Password Flow Authentication
+CALLBACK_AUTHENTICATION = env("CALLBACK_AUTHENTICATION", default="")
+CLIENT_ID = env("CLIENT_ID", default="")
+CLIENT_SECRET = env("CLIENT_SECRET", default="")
+USER_NAME = CLIENT_ID
+PASSWORD = CLIENT_SECRET
+
+OAUTH_TOKEN_URL = env("OAUTH_TOKEN_URL", default="")
+CERTS_DIR = path.join(CONFIG_DIR, "certs")
+# Sensor certificate with private key used as client cert
+OAUTH_PATH_TO_CLIENT_CERT = path.join(CERTS_DIR, env("OAUTH_CLIENT_CERT", default=""))
+# Authserver and Scos-manager server certificate used to verify ssl like truststore if CALLBACK_SSL_VERIFICATION=true
```

Comment:
Recommend to change this comment to something like "Trusted Certificate Authority certificate to verify authserver and callback URL server certificate"